com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/ReflectionMenuItem.cs

```diff
@@ -18,9 +18,7 @@ static void CreateMirrorGameObject(MenuCommand menuCommand)
             var planarProbe = plane.AddComponent<PlanarReflectionProbe>();
             planarProbe.influenceVolume.boxSize = new Vector3(10, 0.01f, 10);
 
-            var hdrp = HDRenderPipeline.defaultAsset;
-            var material = hdrp != null ? hdrp.GetDefaultMirrorMaterial() : null;
-
+            var material = HDRenderPipeline.currentAsset?.GetDefaultMirrorMaterial();
```

Comment:
So we switch from default (gfx one) to current (potentially quality). Is this ok ?

Reply:
i could actually rewrite that to grab the HDRP Global Settings one!

Reply:
fixed in upcoming commit